jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/MetaInfConfiguration.java

```diff
@@ -398,17 +383,8 @@ else if (LOG.isDebugEnabled())
             //not using caches or not in the cache so check for the resources dir
             if (LOG.isDebugEnabled())
                 LOG.debug("{} META-INF/resources checked", target);
-            if (target.isDirectory())
-            {
-                //TODO think  how to handle an unpacked jar file (eg for osgi)
-                resourcesDir = target.resolve("/META-INF/resources");
-            }
-            else
-            {
-                // Resource represents a packed jar
-                URI uri = target.getURI();
-                resourcesDir = resourceFactory.newResource(URIUtil.uriJarPrefix(uri, "!/META-INF/resources"));
-            }
+
+            resourcesDir = target.resolve("META-INF/resources");
```

Comment:
This requires `target` to be a directory, this should be asserted and javadoc'ed.

Reply:
You still need the `if (target.isDirectory())` don't you?